go/vt/sqlparser/ast_funcs.go

```diff
@@ -282,6 +283,46 @@ func (ct *ColumnType) SQLType() querypb.Type {
 	return sqltypes.Null
 }
 
+// AddQueryHint adds the given string to list of comment.
+// If the list is empty, one will be created containing the query hint.
+// If the list already contains a query hint, the given string will be merged with the existing one.
+// This is done because only one query hint is allowed per query.
+func (node Comments) AddQueryHint(queryHint string) (Comments, error) {
```

Comment:
Here I have assumed that query hints only start by `/*+` (based on MySQL documentation), but I found that we also have `/*vt+` at this day, I am not sure whether I should consider those comments as query hints too.
cc @harshit-gangal @systay

Reply:
we should not pass down /*vt+ comments down. It should be removed after using those hints at vtgate.

Reply:
Got it